dash/components/src/Users/Activity/FeedHeader.tsx

```diff
@@ -18,7 +18,7 @@ const FeedHeader: React.FC<Props> = ({ date, numItems, numDeleted }) => (
         <i className="fa fa-chevron-left mr-2" aria-hidden /> Back
       </Link>
       <h1 className="font-bold text-slate-800 block xs:hidden">
-        {formatDate(date, `short`)}
+        {formatDate(date, `m/d/yyyy`)}
```

Comment:
renamed this format, what used to be `short` is now `m/d/yyyy` because i made a new `short` format that is more human oriented, with the day name, based on parent feedback.